CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
```

Comment:
It is a nice addition but is out of scope for this change, can we please remove it?